sql/moz-fx-data-shared-prod/search_derived/remotesettings_suggestions_v1/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+---
+workgroup_access:
```

Comment:
Let's fill in the friendly_name, description, and owners fields here too. We may have logic that requires them, and they're good minimal documentation to give future users context on what this is.